src/v2i-hub/CARMAStreetsPlugin/manifest.json

```diff
@@ -1,48 +1,64 @@
 {
-	"name":"CARMAStreetsPlugin",
-	"description":"Plugin to communicate with CARMA Streets via Apache Kafka",
-	"version":"@PROJECT_VERSION@",
-	"exeLocation":"/bin/CARMAStreetsPlugin",
-	"coreIpAddr":"127.0.0.1",
-	"corePort":24601,
-	"messageTypes":[
-		{
-			"type":"J2735",
-			"subtype":"TMSG03-P",
-			"description":"..."
-		}
-	],
-	"configuration":[
-		{
-			"key":"receiveTopic",
-			"default":"v2xhub_out",
-			"description":"Apache Kafka topic plugin will receive message from."
-		},
-		{
-			"key":"transmitMobilityOperationTopic",
-			"default":"v2xhub_mobility_operation_in",
-			"description":"Apache Kafka topic plugin will transmit message to."
-		},
-		{
-			"key":"transmitMobilityPathTopic",
-			"default":"v2xhub_mobility_path_in",
-			"description":"Apache Kafka topic plugin will transmit message to."
-		},
-		{
-			"key":"KafkaBrokerIp",
-			"default":"127.0.0.1",
-			"description":"IP of Apache Kafka broker."
-		},
-		{
-			"key":"KafkaBrokerPort",
-			"default":"9092",
-			"description":"Port of Apache Kafka broker."
-		},
-		{
-			"key":"MobilityOperationStrategies",
-			"default":"carma/port_drayage,TruckInspection",
-			"description":"A comma separated list of strategies of MobilityOperation messages to send to CARMA Streets"
-		}
-		
-	]
-}
+    "name": "CARMAStreetsPlugin",
+    "description": "Plugin to communicate with CARMA Streets via Apache Kafka",
+    "version": "@PROJECT_VERSION@",
+    "exeLocation": "/bin/CARMAStreetsPlugin",
+    "coreIpAddr": "127.0.0.1",
+    "corePort": 24601,
+    "messageTypes": [{
+        "type": "J2735",
+        "subtype": "TMSG03-P",
+        "description": "..."
+    }],
+    "configuration": [{
+            "key": "receiveTopic",
```

Comment:
Can we remove this configuration value since we have subscribeToSchedulingPlanTopic?

Reply:
Or is this used for something else

Reply:
I think I made this configuration initially to represent the scheduling topic but I never implemented it. Since you implemented this with a different configuration I think we can remove this

Reply:
@paulbourelly999 It is updated. https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/271/commits/c3f41ed5985f2d95ce5d2544238c0edb4add4a55